system/doc/system_principles/misc.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE chapter SYSTEM "chapter.dtd">
+
+<chapter>
+  <header>
+    <copyright>
+      <year>2018</year>
+      <holder>Ericsson AB. All Rights Reserved.</holder>
+    </copyright>
+    <legalnotice>
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+ 
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    
+    </legalnotice>
+
+    <title>Support, Compatibility, Deprecations, and Removal</title>
+    <prepared></prepared>
+    <responsible></responsible>
+    <docno></docno>
+    <approved></approved>
+    <checked></checked>
+    <date>2018-05-21</date>
+    <rev></rev>
+    <file>misc.xml</file>
+  </header>
+
+  <section>
+    <marker id="supported_releases"/>
+    <title>Supported Releases</title>
+    <p>
+      In general, bugs are only fixed on the latest
+      <seealso marker="versions#releases_and_patches">release</seealso>,
+      and new features are introduced in the upcoming release that is
+      under development. However, when we, due to internal reasons, fix
+      bugs on older releases, these will be available and announced as well.
+    </p>
+    <p>
+      Due to the above, pull requests are only accepted on the
+      <c>maint</c> and the <c>master</c> branches in our
+      <url href="https://github.com/erlang/otp">git repository</url>.
+      The <c>maint</c> branch contains changes planned for the next
+      maintenance patch package on the latest OTP release and the
+      <c>master</c> branch contain changes planned for the upcoming
+      OTP release.
+    </p>
+  </section>
+
+  <section>
+    <marker id="compatibility"/>
+    <title>Compatibility</title>
+    <p>
+      We always strive to remain as compatible as possible
+      even in the cases where we give no compatibility guarantees.
+    </p>
+    <p>
+      Different parts of the system will be handled differently
+      regarding compatibility. The following items describe how
+      different parts of the system are handled.
+    </p>
+    <taglist>
+      <tag>Erlang Distribution</tag>
+      <item>
+	<p>
+	  Erlang nodes can communicate across at least
+	  two preceding and two subsequent releases.
```

Comment:
It is unclear to the uninitiated reader whether 'releases' here includes 'maintenance package' or means 'major releases'. It would be worthwhile to be specific, since it has big implications on time durations

Reply:
I added a section in the versions.xml file about this (part of this commit), which is linked from the misc.xml document, in order to clarify such questions.